var/spack/repos/builtin/packages/py-cleo/package.py

```diff
@@ -12,17 +12,19 @@ class PyCleo(PythonPackage):
     homepage = "https://github.com/sdispater/cleo"
     pypi = "cleo/cleo-0.8.1.tar.gz"
 
+    maintainers("LydDeb")
+
+    version("2.0.1", sha256="eb4b2e1f3063c11085cebe489a6e9124163c226575a3c3be69b2e51af4a15ec5")
+    version("2.0.0", sha256="fbc5cb141cbc31ea8ffd3d5cd67d3b183fa38aa5098fd37e39e9a953a232fda9")
     version("1.0.0a5", sha256="097c9d0e0332fd53cc89fc11eb0a6ba0309e6a3933c08f7b38558555486925d3")
```

Comment:
Do we want to deprecate the 1.0.0a5 alpha release now that the final release is out?

Reply:
Good idea, I add version 1.0.0 and deprecate 1.0.0a5.